ci/incremental/setup_conda_environment.cmd

```diff
@@ -13,7 +13,7 @@ conda list
 @rem Clean up any left-over from a previous build
 conda remove --all -q -y -n %CONDA_ENV%
 @rem Scipy, CFFI, jinja2 and IPython are optional dependencies, but exercised in the test suite
-conda env create -n %CONDA_ENV% --file=ci\azure-windows-%CONDA_PY%.yaml
+conda env create -n %CONDA_ENV% --file=ci\azure-windows-%CONDA_PY%.yml
```

Comment:
why are you changing this? all of the other files are .yaml

Reply:
would rather change the existing .yml -> .yaml

Reply:
> all of the other files are .yaml

The top-level config files are .yml (e.g. .travis.yml)

> would rather change the existing .yml -> .yaml

Sure, will update.